packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -475,19 +530,106 @@ class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
        assert(isDismissible != null),
        assert(enableDrag != null);
 
+  /// A builder for the contents of the sheet.
+  ///
+  /// The bottom sheet will wrap the widget produced by this builder in a
+  /// [Material] widget.
   final WidgetBuilder? builder;
+
+  /// Stores a list of captured [InheritedTheme]s that are wrapped around the
+  /// bottom sheet.
   final CapturedThemes capturedThemes;
+
+  /// Specifies whether this is a route fo a bottom sheet that will utilize
+  /// [DraggableScrollableSheet]. Consider setting this parameter to true
```

Comment:
a new line in between first sentence and second. Our style guide suggest the first paragraph should be a single sentence that gives a brief description.